lib/controller.rb

```diff
@@ -7,5 +7,13 @@
 class WeekThreeAssessment < Sinatra::Base
 
   # Your code will go here.
+get '/' do
+  @name = 'Amanda'
+  erb :index
+end
+
+get '/name/:name' do
+  @user = User.create(name: "#{params[:name]}")
```

Comment:
And then what are we doing? Is there a page to be rendered? Your Scenario don't check for what happens next. Just that a user is created.

Reply:
Yeah? I'm not sure what should happen next - as I read the instructions I should create an user with attribute "name" when I visit '/name/:name'. What am I missing? :)

Reply:
That a page should be rendered.